src/PowerShellEditorServices/Services/Workspace/WorkspaceService.cs

```diff
@@ -293,18 +300,23 @@ public void CloseFile(ScriptFile scriptFile)
         /// <summary>
         /// Gets the workspace-relative path of the given file path.
         /// </summary>
-        /// <param name="filePath">The original full file path.</param>
         /// <returns>A relative file path</returns>
-        public string GetRelativePath(string filePath)
+        public string GetRelativePath(ScriptFile scriptFile)
         {
-            string resolvedPath = filePath;
+            string resolvedPath = scriptFile.FilePath;
 
-            if (!IsPathInMemory(filePath) && !string.IsNullOrEmpty(WorkspacePath))
+            if (!scriptFile.IsInMemory)
```

Comment:
I think `IsPathInMemory` can go away because it's only used in two tests now, and `IsInMemory` is used everywhere else. Not sure why we had two ways to go about that.